src/libraries/Microsoft.PowerFx.Core/Public/Engine.cs

```diff
@@ -45,6 +45,15 @@ public IEnumerable<string> GetAllFunctionNames()
             return Config.GetAllFunctionNames();
         }
 
+        /// <summary>
+        /// Returns the maximum number of recursive function calls.
+        /// </summary>
+        /// <returns></returns>
+        public int GetMaxCallDepth()
```

Comment:
internal

Reply:
Unneeded, recursion depth handled in seperate PR.